### PR TITLE
chore: fix GetRef for some case

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     misc: Fix GetRef ts util can not get correct ref type for some component.      |
| 🇨🇳 Chinese |     misc: 修复 GetRef 工具类型不能正确获得某些组件 ref 类型的问题。   |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by li-jia-nan — DISMISSED** [review]

**Review by li-jia-nan — DISMISSED** [review]

**Comment by zombieJ** [comment]
博客里写了，的确可以加个工具分类：

https://ant-design.antgroup.com/docs/blog/type-util-cn#getprops-%E8%8E%B7%E5%8F%96%E5%B1%9E%E6%80%A7%E5%AE%9A%E4%B9%89

**Review by li-jia-nan — DISMISSED** [review]

File: components/_util/index.en-US.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+cover: https://mdn.alipayobjects.com/huamei_7uahnr/afts/img/A*YSm4RI3iOJ8AAAAAAAAAAAAADrJ8AQ/original
+coverDark: https://mdn.alipayobjects.com/huamei_7uahnr/afts/img/A*03dxS64LxeQAAAAAAAAAAAAADrJ8AQ/original
+demo:
+  cols: 2
+group:
+  title: Other
+  order: 99
+---
+
+工具类用于辅助开发，提供一些常用的工具方法。
```
Comment (li-jia-nan) [inline components/_util/index.en-US.md:13]: 中文

**Review by li-jia-nan — DISMISSED** [review]

File: components/_util/index.en-US.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+category: Components
+title: Util
+cover: https://mdn.alipayobjects.com/huamei_7uahnr/afts/img/A*YSm4RI3iOJ8AAAAAAAAAAAAADrJ8AQ/original
```
Comment (li-jia-nan) [inline components/_util/index.en-US.md:4]: 这个是 Affix 组件的封面图吗？

Comment (zombieJ) [reply components/_util/index.en-US.md:4]: 提醒我了，这个东西没图标。。。

Comment (li-jia-nan) [reply components/_util/index.en-US.md:4]: 找设计大佬要一个吗？

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**